documentation/master-solicitor.asciidoc

```diff
@@ -1669,6 +1669,8 @@ Spring beans implementing this interface will be called at certain points in the
 == Release Notes
 Changes in 1.20.0::
 * https://github.com/devonfw/solicitor/issues/232: Set a standard for ordering LicenseNameMapping rules. Rules with an 'or-later' suffix are put before '-only' rules.
+* https://github.com/devonfw/solicitor/pull/235: Enhanced Curation Template Generation.  when generating a curation template and setting the curationdataselector parameter to "none," curations will be skipped, resulting in a template without applied curations.
```

Comment:
The comment in "application.properties" (in core/src/main/resources as well as in the copy in documentation/files) needs to be extended to describe the special value "none".